InterfaceParser/parsers/json_rpc.py

```diff
@@ -0,0 +1,98 @@
+"""JSON RPC parser.
+
+Contains parser for JSON RPC XML format.
+
+"""
+from parsers.parse_error import ParseError
+from parsers.rpc_base import RPCBase
+
+
+class Parser(RPCBase):
+    """JSON RPC parser."""
+
+    def __init__(self):
+        """Constructor."""
+        super(Parser, self).__init__()
+        self._interface_name = None
+
+    @property
+    def get_version(self):
+        return '1.0.0'
+
+    def _parse_root(self, root):
+        """Parse root XML element.
+
+        This implementation parses root as interfaces element with multiple
+        interfaces in it.
+
+        :param root: ElementTree root Element object
+        """
+
+        self._params = root.attrib
+        self._interface_name = None
+
+        for element in root:
+            if element.tag != "interface":
+                raise ParseError("Subelement '{}' is unexpected in interfaces".format(element.tag))
+
+            if "name" not in element.attrib:
+                raise ParseError("Name is not specified for interface")
+
+            self._interface_name = element.attrib["name"]
+            self._parse_interface(element, self._interface_name + "_")
+
+    def _check_enum_name(self, enum):
+        """ Check enum name.
+
+        This method is called to check whether the newly parsed enum's name
+        conflicts with some predefined enum.
+        As SDLRPCV2 has no predefined enums this implementation does nothing.
```

Comment:
Is this whole method definition a copy paste error?

Reply:
Method should be removed, or the description should be fixed

Reply:
description changed:
As **JSON RPC parser** has no predefined enums this implementation does nothing.